ArchiSteamFarm/Bot.cs

```diff
@@ -1982,7 +1957,12 @@ internal sealed class Bot : IDisposable {
 				return;
 			}
 
-			await HandleMessage(notification.steamid_friend, message).ConfigureAwait(false);
+			string response = await Commands.Parse(this, notification.steamid_friend, message).ConfigureAwait(false);
+			if(response == null) {
+				ArchiLogger.LogNullError(nameof(response));
```

Comment:
Do not blindly copy paste, `LogNullError()` should be used ONLY for UNEXPECTED errors, null response is **entirely** respected when the user doesn't have permission to interact with the bot and therefore generates no response.